### ISSUE TITLE
Build and publish daily binaries

### ISSUE THREAD

**Issue by BigLep** [ipfs/go-ipfs#8250]
**Done criteria:** 
1. go-ipfs binaries are built daily
2. built binaries are published to our distribution channels daily
3. we get automatic notification if this process fails

**Why important:** 
1. Performing this process daily catches issues with our automation so we can fix it before we go to rely on it as part of our doing our full release process.
2. if there is an issue with a go-ipfs release (e.g., memory leak), this provides a way to do a more expedient bisect to determine when the issue was introduced.

**Customer:** primarily the core dev team, but it's possible external users will leverage this as well.

**Notes:** 
1. Design / planning work should be done first and reviewed with the team.  We want to make sure we cover specifics around build names, alarm/failure notification, etc.
2. Basically we want a way where when event X occurs, start doing a release (where event X could be midnight UTC).  This includes building everything in CI and then submitting PR's (ipfs-distribtuions repo, infra for DNS records) and uploading to binaries to where they need to go (GitHub and cluster).

**Comment by BigLep** [comment]
Closing this as a duplicate of https://github.com/ipfs/distributions/issues/363
